src/dvc_render/base.py

```diff
@@ -18,10 +18,10 @@ class Renderer(abc.ABC):
 
     EXTENSIONS: Iterable[str] = {}
 
-    def __init__(self, datapoints: Dict, name: str, **properties):
-        self.datapoints = datapoints
-        self.name = name
-        self.properties = properties
+    def __init__(self, datapoints: List, name: str, **properties):
```

Comment:
```suggestion
    def __init__(self, datapoints: List = None, name: str = None, **properties):
```

As you are already expecting a `NoneType`, this should make it more obvious.